packages/headless/doc-parser/src/entity-builder.ts

```diff
@@ -137,7 +137,14 @@ function getParamDescription(param: Parameter) {
   const description = emitAsTsDoc(nodes as unknown as readonly DocNode[]);
 
   if (!description) {
-    throw new Error(`No description found for param: ${param.name}`);
+    const location = /(^\w+)/.exec(
+      param.parameterTypeExcerpt.tokens[0].text
```

Comment:
Will need defensive code around `param.parameterTypeExcerpt.tokens[0]` I think. It might be an empty array.